src/actions/collapseContext.ts

```diff
@@ -61,12 +62,18 @@ const collapseContext = (state: State, { at }: Options) => {
   }
 
   const thought = getThoughtById(state, head(simplePath))
-  const rankStart = getRankBefore(state, simplePath)
-  const rankIncrement = (thought.rank - rankStart) / children.length
 
   // Find the sort preference, if any
   const parentId = head(rootedParentOf(state, simplePath))
   const sortId = findDescendant(state, head(simplePath), ['=sort'])
+  const isSortedContext = !!sortId || findDescendant(state, head(rootedParentOf(state, simplePath)), ['=sort'])
```

Comment:
You can use [getSortPreference](https://github.com/cybersemics/em/blob/a27f56c62a1a16d0b31435201cd77aed7ccd156e/src/selectors/getSortPreference.ts).